mongodb/activity/queryDocument/metadata.go

```diff
@@ -0,0 +1,90 @@
+package queryDocument
+
+import (
+	"github.com/project-flogo/core/data/coerce"
+)
+
+// Settings structure
+type Settings struct {
+	Connection string `md:"mongoConnection,required"`
+	Operation  string `md:"operation,required"`
+	CollName   string `md:"collName,required"`
+}
+
+//Input structure
+type Input struct {
+	Input map[string]interface{} `md:"input,required"`
+}
+
+//Output structure
+type Output struct {
+	Output interface{} `md:"Output"` //
```

Comment:
Shouldn't it be lowercase `md:"output"` ?

Reply:
will make it lower case output